src/main/java/uk/gov/hmcts/reform/ucmc/model/SolicitorReferences.java

```diff
@@ -0,0 +1,13 @@
+package uk.gov.hmcts.reform.ucmc.model;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import lombok.Builder;
+import lombok.Data;
+
+@Data
+@Builder
+@JsonInclude(JsonInclude.Include.NON_NULL)
+public class SolicitorReferences {
+    private final String solicitorReference;
```

Comment:
I think we have renamed this to `claimantReference`.

Reply:
yeah I've made the mistake of building this off of 353 🤦